doc/user/content/sql/drop-type.md

```diff
@@ -84,6 +84,22 @@ SELECT * FROM t;
 ERROR:  unknown catalog item 't'
 ```
 
+### Remove a type only if it has no dependent objects
```

Comment:
Just seconding the suggestion here to make clear that you are not showing sequential commands you could run like the examples elsewhere.